frontend/packages/dev-console/src/components/BuildConfigPage.tsx

```diff
@@ -1,30 +1,25 @@
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
-import { match as RMatch } from 'react-router';
 import { BuildConfigsPage } from '@console/internal/components/build-config';
+import { withStartGuide } from '../../../../public/components/start-guide';
 import DefaultPage from './DefaultPage';
 
-export interface BuildConfigPageProps {
```

Comment:
Let's not remove this part. Keep the component `BuildConfigPage` with the correct defined prop type. Then add the `withStartGuide` HOC. Otherwise we lose out on the typing of `props`